compact/tracking/definitions_craterlake.xml

```diff
@@ -52,40 +61,42 @@
     <constant name="ForwardTOF_zmin"              value="ForwardTOFRegion_zmin + ForwardTOFRegion_length - ForwardTOF_length - 1*cm"/>
 
     <comment> Service/Support setup </comment>
-    <constant name="VertexSupportCyl2_thickness"     value="0.3*cm"/>
-    <constant name="VertexSupportCylCF_thickness"    value="100.0*um" />
-    <constant name="TrackerSupportCyl_rmin"          value="44.0*cm"/>
+    <!-- <constant name="VertexSupportCyl2_thickness"     value="0.3*cm"/> -->
+    <!-- <constant name="VertexSupportCylCF_thickness"    value="100.0*um" /> -->
+    <!-- <constant name="TrackerSupportCyl_rmin1"          value="TrackerEndcapDisk_rmax + 0.5*cm"/> -->
+    <constant name="TrackerSupportCyl_rmin1"          value="InnerMPGDBarrel_rmin - 2*cm"/>
+    <constant name="TrackerSupportCyl_rmin2"          value="InnerMPGDBarrel_rmin + 2*cm"/>
     <comment> These may belong in definitions.xml </comment>
     <constant name="InnerTrackerEndcapP_zmax"        value="CentralTrackingRegionP_zmax"/>
     <constant name="InnerTrackerEndcapN_zmax"        value="CentralTrackingRegionN_zmax"/>
-    <comment> Endcap structures </comment>
+<!--     <comment> Endcap structures </comment>
```

Comment:
Remove commented lines, and definitely don't use mulitline `<!--` since they cannot be nested.